rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_reader :book, :person
+  attr_accessor :date
+
+  def initialize(date, book, person)
+    @date = date
+
+    @book = book
+    book.retals << self
+
+    @person = person
+    person.retals << self
```

Comment:
I think you have some typo error with the `rentals`, Please fix this 👍

Reply:
thanks for that I didn't notice the typo error 😄